### DISCUSSION TITLE
Multiple Route Navigation

### DISCUSSION THREAD

**Discussion by pakitaleo** [react-navigation/react-navigation#12249] [Q&A]
So i want to create a multiple rout navigation, as of there is a list of screen to navigate

im creating a wine app and i want to know how do i sort the navigation history so i dont need to create redundance screens, as the user can select a wine from a country then select the grape type then select the brand of the wine and vice-versa, any ideas how i can do that?

**Comment by satya164** [comment]
What's multiple route navigation?

If you want each filter to have only one screen in history, you can use `getId` https://reactnavigation.org/docs/screen/#id
